doc/contributing/docs/markup/tables.rst

```diff
@@ -5,15 +5,15 @@ Tables are very useful and rST markup
 `offers <https://docutils.sourceforge.io/docs/ref/rst/directives.html#tables>`_
 different ways to create them.
 
-We prefer list-tables to create table of contents:
+We prefer list-tables because they allow you to put as much content as you need
+without painting ASCII-art borders:
 
-..  code-block:: rst
+..  literalinclude:: _includes/table.rst
+    :language: rst
 
-    ..  container:: table
+And this table will appear like this:
```

Comment:
The idea this sentence expresses is very clear, yet I would suggest to rephrase it in order to avoid double "this", for instance "_This is how the table will look like:_".